src/LoginServer/Database/LoginDb.cs

```diff
@@ -155,16 +155,8 @@ public void CreateCharacter(long accountId, Character character)
 					cmd.Set("bz", character.BarrackPosition.Z);
 
 					cmd.Set("hp", character.Hp);
-					cmd.Set("maxHp", character.MaxHp);
 					cmd.Set("sp", character.Sp);
-					cmd.Set("maxSp", character.MaxSp);
 					cmd.Set("stamina", character.Stamina);
-					cmd.Set("maxStamina", character.MaxStamina);
-					cmd.Set("str", character.Str);
```

Comment:
I think this change is problematic. I get the idea, and it might be what officials do, but sending a job stat that comes from the static data instead of a "free flowing" value is *highly* inflexible. Getting to 1 on each stat with these changes would basically require hacks, but that will be something that users will probably want, to allow completely free stat distribution.